docs/hermes.md

```diff
@@ -52,3 +52,23 @@ $ react-native run-android --variant release
 ```
 
 This will compile JavaScript to bytecode during build time which will improve your app's startup speed on device.
+
+## Debugging Hermes using Google Chrome's DevTools
+
+Hermes supports the Chrome debugger by implementing the Chrome inspector protocol. This means Chrome's tools can be used to directly debug JavaScript running on Hermes, on an emulator or device.
+
+Chrome connects to Hermes running on device via Metro, so you'll need to know where Metro is listening. Typically this will be on `localhost:8081`, but this is [configurable](https://facebook.github.io/metro/docs/en/configuration). When running `yarn start` the address is written to stdout on startup.
+
+Once you know where the Metro server is listening, you can connect with Chrome using the following steps:
+
+1. Navigate to `chrome://inspect` in a Chrome browser instance.
+
+2. Use the `Configure...` button to add the Metro server address. E.g.:
```

Comment:
Might as well mention that the metro server address is generally `localhost:8081`

Reply:
That is mentioned above... although I guess I could repeat it here.